packages/suite/src/views/wallet/staking/components/StakingDashboard/StakingDashboard.tsx

```diff
@@ -17,16 +17,20 @@ export const StakingDashboard = ({ selectedAccount, dashboard }: StakingDashboar
         selectAccountHasStaked(state, selectedAccount?.account?.key ?? ''),
     );
 
-    if (!selectedAccount) return null;
```

Comment:
if you check `selectedAccount.status !== 'loaded'` you then dont need `?` in `titleValues`